src/runtime_src/core/edge/drm/zocl/zocl_ioctl.c

```diff
@@ -199,6 +199,7 @@ zocl_fpga_mgr_load(struct drm_device *ddev, char *data, int size)
 {
 	struct drm_zocl_dev *zdev = ddev->dev_private;
 	struct device *dev = ddev->dev;
+	/* zocl_pcap_download_ioctl() has checked if fpga_mgr is NULL */
```

Comment:
Looks like the better place to check if fpga_mgr is NULL is inside this function, because except pcap_download, pdi download also make use of this function

Reply:
Will do.